rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/bulk/12_noop.yml

```diff
@@ -0,0 +1,35 @@
+---
+"Noop":
+  - skip:
+      version: " - 8.13.99"
+      reason: fixed in 8.14.0
+  - do:
+      indices.create:
+        index: test_1
+        body:
+          settings:
+            number_of_shards: 1
+  - do:
+      bulk:
+        refresh: true
+        body: |
+          { "create": { "_index": "test_1", "_id": "1"} }
+          { "foo": "bar" }
+          { "create": { "_index": "test_1", "_id": "2"} }
+          { "foo": "bar" }
+  - do:
```

Comment:
I think it makes sense to check the index stats after the first bulk requests to make sure that it was not a no-op update
```
  - do:
      indices.stats: { index: test_1 }

  - match: { _all.total.indexing.index_total: 2 }
  - match: { _all.total.indexing.noop_update_total: 0 }
```

Reply:
Thanks for the review! I've made the suggested changes.